lib/model/frame.js

```diff
@@ -26,6 +26,10 @@ const table = (name, to) => (def) => {
   def.from = name;
   def.to = to || name.substring(0, name.length - 1); // take off the s
 };
+const fieldTypes = (types) => (def) => {
+  def.fieldTypes = types;
+};
+
```

Comment:
Being pragmatic here, don't want to change a whole lot of things. With this function, we can define fieldTypes for only those frames that are bulk inserted using `insertMany` function.

Alternative was to change the def.fields from array of string to array of object with name, type, readable, writeable, etc or have associated array of types and define type for all fields in every frame.

Reply:
we can remove this when and if https://github.com/gajus/slonik/issues/455 is implemented